robyn/__init__.py

```diff
@@ -302,110 +310,193 @@ def inner(handler):
 
         return inner
 
-    def get(self, endpoint: str, const: bool = False, auth_required: bool = False):
+    def get(
+        self,
+        endpoint: str,
+        const: bool = False,
+        auth_required: bool = False,
+        openapi_tags: list = ["default"],
```

Comment:
this should default to 
```suggestion
        openapi_tags: list = ["get"],
```